afids_cnn/train.py

```diff
@@ -6,11 +6,13 @@
 
 import numpy as np
 import pandas as pd
+import tensorflow as tf
 from numpy.typing import NDArray
 from tensorflow import keras
```

Comment:
This gets referred to a lot through `tf.losses`, `tf.metrics`, etc. so I think we still need it, but may be wrong

Reply:
Hmm, that feels like that should be handled in the backend by `tf`, rather than requiring an additional import on the end-user.

In any case, I was wrong and this import is still needed as a lot of the typehints use `keras.x` in the output. Alternatively, can change those to `tf.keras.x`

Reply:
Ok I'll try to get rid of it